packages/react/src/components/statusLabel/StatusLabel.tsx

```diff
@@ -1,5 +1,8 @@
 import React from 'react';
-
+/**
+ * Note: default rectangular StatusLabel will be deprecated in 3.0.0. 
+ * Rounded corder variant will be used as default instead.
+ */
```

Comment:
If you move this JSDOC right before the export const StatusLabel, it will show to users and be very informative.